Package.resolved

```diff
@@ -348,8 +348,8 @@
       "kind" : "remoteSourceControl",
       "location" : "https://github.com/tuist/XcodeGraph.git",
       "state" : {
-        "revision" : "decf6cc311e8dd238c87501c369b05827bca6a48",
-        "version" : "1.5.14"
+        "branch" : "feat/local-package-mapping",
+        "revision" : "7c326d652b1177b099c5abe890bb864d1a73be41"
```

Comment:
todo: revert this to a proper release once https://github.com/tuist/XcodeGraph/pull/116 is merged